### DISCUSSION TITLE
XTTS V2 full training pipeline for all stages

### DISCUSSION THREAD

**Discussion by smallsudarshan** [idiap/coqui-ai-TTS#16] [General]
**🚀 Feature Description**

I was working on training XTTS and had created [this](https://github.com/coqui-ai/TTS/issues/3704) issue in the original repo.

Is this the right way to solve the problem? Would be great to get some direction/validation of the problem statement.

**Comment by eginhard** [comment]
Sorry @smallsudarshan, I'm not affiliated with the original Coqui developers and don't have insights into the XTTS training process.

**Reply by smallsudarshan** [reply]
Ah no worries @eginhard. Would this be a value addition to this repository in case we are able to fix it? Or is this maybe something different from the repo roadmap (in which case, we will continue working in the original repo)?

**Reply by eginhard** [reply]
I won't say no categorically, but I'd say it's unlikely that we'd merge something like this for now if it's not a full reproduction of XTTS on a public dataset with a training recipe and model release. But you're still welcome to submit a PR of course to share some partial solutions with others. We'll definitely merge any XTTS-related bug fixes like #4 though.
